bin/tests/it/k8s.rs

```diff
@@ -28,7 +28,7 @@ async fn print_pod_logs(client: Client, namespace: &str, label: &str) {
             async move {
                 let mut logs = pods
                     .log_stream(
-                        &p.name(),
+                        &p.metadata.name.clone().unwrap(),
```

Comment:
always Some()?

Reply:
Not sure, with the update to our rust dependencies, these values are now returned as an Option. IMO given that it's a test I think it's okay to just unwrap without trying to handle the error - panicking will be seen as a test failure.